app/views/content_items/detailed_guide.html.erb

```diff
@@ -6,15 +6,26 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+    <% title_and_context = @content_item.brexit_hub_page? ? { title: @content_item.title } : @content_item.title_and_context %>
+    <%= render 'govuk_publishing_components/components/title', title_and_context %>
   </div>
   <%= render 'shared/translations' %>
   <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+    <% if @content_item.brexit_hub_page? %>
+      <div class="govuk-body-l govuk-!-margin-bottom-7">
+        <% link_text = @content_item.brexit_citizen_page? ? "Brexit guidance for businesses" : "Brexit guidance for individuals and families" %>
+        <% link_path = @content_item.brexit_citizen_page? ? @content_item.brexit_business_page : @content_item.brexit_citizen_page %>
+        <p class="govuk-body">
+          There's different <%= link_to link_text, link_path %>.
```

Comment:
```suggestion
          There's different <%= link_to link_text, link_path, class: "govuk-link" %>.
```

otherwise the hover state will use the old, less-accessible, orange styles